drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -79,5 +79,15 @@ config ETH_STM32_HAL_MAC5
 	  This is the byte 5 of the MAC address.
 endif
 
+choice
+        prompt "MII/RMII Interface to the Physical Layer"
```

Comment:
need a line "default ETH_STM32_HAL_MII"

However, why having a choice if only one of these 2 config is actually used in the code?
You could have a config ETH_STM32_HAL_RMII "Swtich to RMII instead of default MII"  or something like that.

Reply:
copied this from drivers/ethernet/Kconfig.sam_gmac

Reply:
Changed the setting to a single bool. It really looks seems more plausible this way.
Thanks for your comment.